chain/scripts/start-rollup.sh

```diff
@@ -45,6 +45,11 @@ sed -i'.bak' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' /root/.world/co
 
 sed -i '/api\]/,/\[/ s/enable = false/enable = true/' /root/.world/config/app.toml
 
+# Cosmos SDK gRPC listener
+sed -i'.bak' 's#"localhost:9090"#"0.0.0.0:9090"#g' /root/.world/config/app.toml
+# Cosmos SDK API server listener
+sed -i'.bak' 's#"localhost:1317"#"0.0.0.0:1317"#g' /root/.world/config/app.toml
```

Comment:
I put an extra double quote here before; just realized it when I deployed it previously.
```suggestion
sed -i'.bak' 's#localhost:1317#0.0.0.0:1317#g' /root/.world/config/app.toml
```